_includes/head.html

```diff
@@ -14,7 +14,21 @@
     <link rel="mask-icon" sizes="any" href="/images/opensourcedesign-logo.svg" color="#000">
 
     <link href="/css/main.css" rel="stylesheet">
-
+    
+    <!-- Facebook metadata -->
+    <meta property="og:url" content="https://opensourcedesign.net">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Open Source Design">
+    <meta property="og:image" content="https://opensourcedesign.net/images/opensourcedesign-logo.png">
+    <meta property="og:description" content="Connecting open source & design! 🎨">
+    <meta property="og:site_name" content="Open Source Design">
+    <meta property="og:locale" content="en_US">
+    
+    <!-- Twitter metadata -->
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:url" content="https://opensourcedesign.net">
+    <meta name="twitter:description" content="Connecting open source & design! 🎨">
+    
```

Comment:
Ideally the meta tags should also close. You can use the "/>" or "</meta>", in order for the HTML to be strict valid.

Reply:
We could also add "twitter:site" since we have an active Twitter account https://twitter.com/opensrcdesign 
Compared for example to our dead Facebook account.

Reply:
"twitter:url" doesn't exists.

Reply:
After merging, we should use "https://cards-dev.twitter.com/validator" to validate

Reply:
Actually we can already preview it using the "https://raw.githubusercontent.com/opensourcedesign/opensourcedesign.github.io/66209de1e75aba48ec18f86008932855a118ab6c/_includes/head.html" URL in
* https://cards-dev.twitter.com/validator
* https://developers.facebook.com/tools/debug/

For the Facebook debugger these are the following warnings: 
"Inferred Property
The 'og:image' property should be explicitly provided, even if a value can be inferred from other tags.
Missing Properties
The following required properties are missing: og:url, og:type, og:title, og:image, og:description, fb:app_id
"

Reply:
Alright thanks.  I'll work on this now.

Reply:
What do I put for FB app ID?

Reply:
We have retired our FB, so we can skip that properties. The validator will provide a warning for the missing field.

Reply:
Got it

Reply:
Looks ready for merge now.  Fixed everything.